applications/machine_learning/sample.yaml

```diff
@@ -7,12 +7,13 @@ tests:
     build_only: true
     platform_allow: >
       nrf52840dk/nrf52840 thingy53/nrf5340/cpuapp thingy53/nrf5340/cpuapp/ns
-      nrf5340dk/nrf5340/cpuapp
+      nrf5340dk/nrf5340/cpuapp nrf54l15dk/nrf54l15/cpuapp
```

Comment:
Can we add it in the separate line? It makes it easier to solve conflicts and is easier to read.

Reply:
Yes. Done.